### PR TITLE
Monitoring: Update monitoring charts to get colors from color studio

### PR DESCRIPTION
**Pull request by matt-west**
Fixes https://github.com/Automattic/dotcom-forge/issues/9474

## Proposed Changes

* Update the color used by the charts on the Monitoring page to use our latest color palette.
* Pull colors from the color-studio package so they are kept in-sync with future color updates.
* Uses the `chroma` package to automatically generate transparent colors, future-proofing against future changes to the color-studio package.

Left: Before | Right: After
 
Screenshot 2024-10-21 at 12 59 48

## Why are these changes being made?

* To ensure colors used on the Monitoring page match our brand palette.

## Testing Instructions

* Go to `/site-monitoring/[domain]`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -260,7 +261,7 @@
 		"bunyan": "^1.8.15",
 		"chalk": "^4.1.2",
 		"check-node-version": "^4.0.2",
-		"chroma-js": "^2.1.2",
+		"chroma-js": "^2.6.0",
```
Comment (sejas) [inline package.json:264]: I'm seeing chroma-js is imported as `devDependencies`. They only other place it's used is in NodeJS:

https://github.com/Automattic/wp-calypso/blob/a54137c0162ff2adfd92239ec20b6f92e11110f3/bin/audit-svg-colors.js#L18

**Review by wojtekn — APPROVED** [review]
The change looks clear, and works as expected.

**Review by sejas — COMMENTED** [review]
I confirm I see the new colors.
I'm unsure of using `chroma-js`,[ it adds `~14102 bytes added 📈 [gzipped]` ](https://github.com/Automattic/wp-calypso/pull/95550#issuecomment-2426439809) to the loaded JS .

@matt-west, what do you think about using regular constants for those colors?

Screenshot 2024-10-23 at 11 00 03

If we decide that loading that library is worth it for this use case, then we should move it to `dependencies` in `pacakge.json`.

**Comment by sejas** [comment]
We can consider creating a follow-up PR and undoing the changes about `chroma-js`.

**Comment by matt-west** [comment]
@sejas I decided to use `chroma-js` here so that we can continue to pull colors from the colour-studio package. I’d rather not hard-code them as it’s easy for them to get outdated again.

Ideally we’d have a function in the colour-studio package itself to generate alpha hex codes, but we’re not quite there yet.

I'll create a follow-up that uses a different approach.

**Comment by matt-west** [comment]
@sejas I’ve opened #95614 to remove `chroma-js`.

**Pull request merged**